hyperjumptech/monika
# Create deployment pipeline for Homebrew

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Add additional github action to publish package to [Homebrew Core Repository](https://github.com/Homebrew/homebrew-core) so monika can be installed via Homebrew
2. Fix typos
3. Add example for monika.rb file

## How did you implement / how did you fix it  
1. Adding publish-homebrew.yml

## How to test  
1. In Actions menu, there will be "Publish to Homebrew" action. it can be run on release or manually
2. After it run, check the [Homebrew Core Repository PR list](https://github.com/Homebrew/homebrew-core/pulls), there will be a new PR from monika
